summingbird-online/src/main/scala/com/twitter/summingbird/online/option/AllOpts.scala

```diff
@@ -98,8 +99,22 @@ trait SummerBuilder extends Serializable {
 
 /**
  * The SummerConstructor option, set this instead of CacheSize, AsyncPoolSize, etc.. to provide how to construct the aggregation for this bolt
+ * @see [[Summers]] for useful [[SummerWithCountersBuilder]]s.
  */
-case class SummerConstructor(get: SummerBuilder)
+case class SummerConstructor(get: SummerWithCountersBuilder)
+
+trait SummerWithCountersBuilder {
```

Comment:
Actually I prefer to keep it non serializable, this code (creating actual `SummerBuilder` with `Counter`s) should happen only on submitter node, and `SummerBuilder` is an object which should be serializable (and it is).